src/main/java/com/cleanroommc/groovyscript/compat/mods/theaurorian/MoonlightForge.java

```diff
@@ -41,10 +41,10 @@ public boolean remove(MoonlightForgeRecipe recipe) {
         });
     }
 
-    @MethodDescription(example = @Example("item('theaurorian:aurorianiteingot')"))
-    public boolean removeByInput(IIngredient output) {
+    @MethodDescription(example = @Example("item('theaurorian:moonstonesword'), item('theaurorian:aurorianiteingot')"))
+    public boolean removeByInput(IIngredient input, IIngredient catalyst) {

```

Comment:
should give this a specific lang key, currently it will be using the general `groovyscript.wiki.removeByInput`. theres technically a `groovyscript.wiki.removeByInputAndCatalyst` thats used, but its used all of one place, so i would suggest using a custom lang key description here.

Reply:
![Screenshot_2024-07-04-10-18-59-499_com.android.chrome-edit.jpg](https://github.com/CleanroomMC/GroovyScript/assets/52964431/56cf579a-26f6-4720-afc6-3d97375e72a9)

Reply:
done

Reply:
actually, how viable would doing this be?
```java
    @MethodDescription(example = @Example("item('theaurorian:moonstonesword')")
    public boolean removeByInput(IIngredient input) {
        return MoonlightForgeRecipeHandler.allRecipes.removeIf(r -> {
            if (input.test(r.getInput1()) || input.test(r.getInput2())) {
                addBackup(r);
                return true;
            }
            return false;
        });
```

Reply:
Thats how it was before? I think if it has to inputs you should supply 2 inputs to match for.

Reply:
That's literally what it was before my changes in the last 2 days. I can roll back if needed.